.gitignore

```diff
@@ -12,6 +12,8 @@ dist
 dist-ssr
 *.local
 
+.env
```

Comment:
Minor suggestion: would be good to add an example.env with dummy values like:

```env
DB_USERNAME=username
DB_PASSWORD=password
```

So that for others setting this project up who do not have access to the `.env`, they know what variables they'll need to obtain.